src/OrchardCore/OrchardCore.AsModule/OrchardCore.AsModule.props

```diff
@@ -8,6 +8,7 @@
   <PropertyGroup>
     <ModuleManifest>Name: $(MSBuildProjectName)</ModuleManifest>
     <ExcludedFiles>**\*.cs;*.csproj*;obj\**;bin\**;Assets.json;Assets\**;Properties\**;*.props;*.targets;.vs\**</ExcludedFiles>
+    <Temp Condition="'$(Temp)' == ''">..\..\..\temp</Temp>
   </PropertyGroup>
```

Comment:
AsModule is also provided as a package which embeds this .props file. So, any user project and at any place can reference this package and then use this .props file.

So, here the problem i see is to use a path relative to the project being built, we would need an absolute path. Maybe with something like `$(SolutionDir)`, but through the command line this one is `Undefined` ...